### ISSUE TITLE
Benchmark results not consistent ?

### ISSUE THREAD

**Issue by hasangenc0** [Acanguven/rapidash#29]
When I write my solution on the top of the other solutions the winner becomes my solution.

Code                                   |  Result
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/26026780/74083789-fed31a00-4a78-11ea-81ff-0f6043b6b27e.png)  |  ![](https://user-images.githubusercontent.com/26026780/74083847-8d479b80-4a79-11ea-8c87-196437e0870d.png)

But if I write my solution after others the winner is changing.

Code                                   |  Result
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/26026780/74083888-df88bc80-4a79-11ea-9658-a8ba90626bce.png)  |  ![](https://user-images.githubusercontent.com/26026780/74083890-e4e60700-4a79-11ea-8afe-57a862d5f0d2.png)

Even if I write my solution twice the results are different.

Code                                   |  Result
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/26026780/74084038-c123c080-4a7b-11ea-8f5c-29de35b73464.png)  |  ![](https://user-images.githubusercontent.com/26026780/74084096-6a6ab680-4a7c-11ea-8f9b-17ced40647d1.png)

The results actually can be normal because of the behavior of this kind of performance analysis. I'm just wondering these tests are reliable.
Maybe the usage of `Benchmark.prototype.hz` is wrong 🤔

**Comment by yavuzkoca** [comment]
Actually its quite normal since the solutions are pretty close to each other. However, since the solutions are tested in dockerized environment it would minimize the difference

**Comment by hasangenc0** [comment]
It is right but the problem is when I right my solution at the top "always" my solution wins and if I write the bottom the winner is changing. And the difference between these even between the duplicate solutions is 2x.

**Comment by yavuzkoca** [comment]
I don't think that it would depend order of the solutions but maybe there is a glitch

**Comment by Acanguven** [comment]
I am not sure if this is a bug, I will investigate this.

**Comment by webbrain-one** [comment]
Hi! Thanks for reporting this. I've opened a PR to address the benchmark inconsistency you noticed. The fix resolves the issue where results were changing based on the order of execution, ensuring more reliable performance metrics. You can review the changes here: https://github.com/Acanguven/rapidash/pull/56. Let me know what you think!
